paper-slider.html

```diff
@@ -437,13 +481,18 @@
       this.fire('change');
     },
 
-    _decrementKey: function(ev, keys) {
-      if (keys.key === 'home') {
+    _decrementKey: function(event) {
+      if (event.detail.key === 'home') {
         this.value = this.min;
       } else {
         this.decrement();
       }
       this.fire('change');
+    },
+
+    _focusChanged: function(focus) {
+      this.$.ink.hidden = !focus;
+      this.$.ink.holdDown = focus;
```

Comment:
I think we might be duplicating some behavior here: https://github.com/PolymerElements/paper-behaviors/blob/master/paper-radio-button-behavior.html

@notwaldorf proposed that `paper-radio-button-behavior` be renamed `paper-inky-behavior` or something to that effect. WDYT?